### ISSUE TITLE
Reason for updating ArcFace implementation

### ISSUE THREAD

**Issue by gau-nernst** [deepinsight/insightface#2218]
I'm referring to commit 657ae30e

In the past, there used to be a threshold that if the logits exceed that threshold, another value is used instead.

**Comment by gau-nernst** [comment]
Thank you for your reply. How about the deletion of this condition?

https://github.com/deepinsight/insightface/commit/657ae30e41fc53641a50a68694009d0530d9f6b3#diff-72f374fed7f382c297361436ed145fdcfd3df633d3bcdb08777ceaab7ea06696L86-L87

From what I gathered from other related issues, this condition was to ensure that the modification to the logits is a monotonic transform.

https://github.com/deepinsight/insightface/issues/2126

Although now the implementation is closer to the paper, in the past ArcFace code has always had this condition. So I wonder how this sudden change came about.

**Comment by nttstar** [comment]
removing the condition doesn't change the result
